seos_native_peripheral.c

```diff
@@ -297,29 +301,47 @@ void seos_native_peripheral_process_message_cred(
 void seos_native_peripheral_process_message_reader(
     SeosNativePeripheral* seos_native_peripheral,
     NativePeripheralMessage message) {
-    BitBuffer* response = bit_buffer_alloc(128); // TODO: MTU
 
-    uint8_t* data = message.buf;
-    uint8_t* rx_data = data + 1; // Match name to nfc version for easier copying
+    uint8_t flags = message.buf[0];
 
-    if(data[0] != BLE_START && data[0] != 0xe1) {
```

Comment:
If we're dropping the use of BLE_START, should there be a line in the diff to delete it?  I love having more semantic parsing of the byte (flags TIL!), but we do want more semantic names for the flags/masks (#define probably)